provider/parameter_store_provider.go

```diff
@@ -33,11 +31,15 @@ type ParameterStoreProvider struct {
 	clients []ParameterStoreClient
 }
 
+type ParameterStoreGetter interface {
```

Comment:
Since there is no longer an `ssmiface` package, we just create an interface for the one call we need to make